### ISSUE TITLE
mise en forme de tableaux excel

### ISSUE THREAD

**Issue by emilienschultz** [emilienschultz/pyshs-bib#38]
Il faudrait avoir une api minimale pour la mise en forme jolie de tableaux excels pour éviter de tout faire à la main

**Comment by jeanbaptisteb** [comment]
D'expérience, cela peut être **très** casse-tête à faire à partir de pandas. De mémoire, il y a aussi quelques impossibilités techniques qui limitent ce qu'on peut faire. Je peux y jeter un oeil (je dois sans doute avoir du code utilisé dans d'autres projets privés, qui peut sans doute être partiellement réutilisé ici). Est-ce qu'il y a un exemple "standard" de tableau qu'on souhaiterait obtenir ?

**Comment by emilienschultz** [comment]
Non, c'est une issue encore très peu définie. A minima, pour les régressions, une présentation un peu standard. L'enjeu est qu'on utilise souvent des sorties intermédiaires qu'ensuite on finalise sous Excel et co. L'idée serait de faire déjà une partie de cette mise en forme : taille des cellules bien dimensionnée ; rajouter des métadonnées ; c'était une issue ouverte pour garder ça en tête (a minima développer la fonction vers_excel pour mettre tous les tableaux intermédiaires dans un même document en la rendant plus souple)
